### ISSUE TITLE
Time-spinner eating

### ISSUE THREAD

**Issue by mikebryant** [mapledyne/yaaz#67]
Currently disabled.
Decide when this is optimal, and do it

**Comment by mapledyne** [comment]
Yeah - I had it in there but it was seeming that remembering past fights was almost always better than the food being remembered, which is why I commented it out.

Logic needs to go something like:
Find difference in food we can already eat vs food we can remember (remembering good food isn't that helpful if we have "pretty good food" in inventory)
Compare with estimated turn savings from remembering a fight that can earn us quest progress.

In looking at other people's ascension logs, it seemed like almost no one was remembering food (hence the disabling until more nuance could be figured out) in favor of remembering fights.

Still, I'm sure there are cases (particularly on the first day?) where food>fight with the spinner.
